From e1a2aad4d130e7a2be38e93d51cdf0793df4933d Mon Sep 17 00:00:00 2001
From: Sina Hatamian <sina.hatamian75@gmail.com>
Date: Sat, 23 May 2026 10:42:59 +0330
Subject: [PATCH 38/89] style: copyright text color change

---
 resources/js/components/HomeFooter.vue | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/resources/js/components/HomeFooter.vue b/resources/js/components/HomeFooter.vue
index 4231c21..7efc050 100644
--- a/resources/js/components/HomeFooter.vue
+++ b/resources/js/components/HomeFooter.vue
@@ -150,7 +150,7 @@ const headingClass =
                 </div>
 
                 <p
-                    class="mt-16 text-center text-[14px] font-normal leading-normal tracking-[-0.56px] text-primary sm:mt-20 lg:mt-[128px]"
+                    class="mt-16 text-center text-[14px] font-normal leading-normal tracking-[-0.56px] text-[#8e8e8e] sm:mt-20 lg:mt-[128px]"
                 >
                     © Copyright 2026. Lorespinner Inc. All rights reserved.
                 </p>
-- 
2.38.1

