From 2d62d003dffcfebd6c3ba0d43c80bd04860e8f2c Mon Sep 17 00:00:00 2001
From: Sina Hatamian <sina.hatamian75@gmail.com>
Date: Tue, 26 May 2026 11:10:15 +0330
Subject: [PATCH 55/89] chan: account to login

---
 resources/js/components/HomeHeaderProfile.vue | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/resources/js/components/HomeHeaderProfile.vue b/resources/js/components/HomeHeaderProfile.vue
index 7c59727..6381a87 100644
--- a/resources/js/components/HomeHeaderProfile.vue
+++ b/resources/js/components/HomeHeaderProfile.vue
@@ -19,7 +19,7 @@ const profileDrawerVisibility = ref(false);
             :href="login.create().url"
             class="flex h-10 items-center justify-center rounded-full border border-[#00C6DE]/70 bg-transparent px-4 text-sm font-medium text-[#c8ced1] transition-colors hover:border-[#00C6DE] hover:text-white"
         >
-            Account
+            Login/Sign Up
         </Link>
     </template>
     <template v-else>
-- 
2.38.1

